src/library/playlisttablemodel.cpp

```diff
@@ -319,6 +319,22 @@ void PlaylistTableModel::shuffleTracks(const QModelIndexList& shuffle, const QMo
     m_pTrackCollectionManager->internalCollection()->getPlaylistDAO().shuffleTracks(m_iPlaylistId, positions, allIds);
 }
 
+double PlaylistTableModel::getDurationOfRows(const QModelIndexList& indices) {
```

Comment:
can you add a short explanatory comment, like:
// Derive total duration from the display widget instead of the database to speed up calculation.

Reply:
I'll add that in DlgAutoDJ where it is called, no need to mention that here IMO